requirements.txt

```diff
@@ -5,5 +5,5 @@ canonicalwebteam.discourse==4.0.8
 canonicalwebteam.search==1.0.0
 canonicalwebteam.templatefinder==1.0.0
 canonicalwebteam.image-template==1.3.1
-black==20.8b1
+black==22.3.0
```

Comment:
Do we need `flake8` and `black`? we don't usually include python linting modules